src/pages/patterns/loading-pattern/index.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Loading
+---
+
+<PageDescription>
+
+Loading patterns are used when information takes an extended amount of time to process and appear on screen. Skeleton states and the loading component are two interactions that communicate to users that data is loading and the screen is not frozen.
+
+</PageDescription>
+
+## Skeleton states
+
+Skeleton states are simplified versions of components used on an initial page load to indicate that the information on the page has not fully loaded yet. They should only appear for one to three seconds, disappearing once the actual content populates the page.
+
+A skeleton state is a stripped-down version of a component, using a `field-01` rectangular block for text. Motion is used in a skeleton state to convey that the page is not stuck loading but rather that data is still being pulled in.
+
+![Example of a text skeleton state in a data table](images/skeleton.png)
+
+<Caption>Example of a text skeleton state in a data table</Caption>
+
+### Usage
+
+Only use skeleton states on container-based components (e.g. tiles and structured lists), data-based components (e.g. data tables and cards), and data-based text. In most cases, action components (e.g. buttons, input fields, checkboxes, toggles) do not need to have a skeleton state.
+
+Never represent toast notifications, overflow menus, dropdown items, modals, and loaders with skeleton states. Elements inside a modal may have a skeleton state, but the modal itself should not.
+
+## Progressive loading
+
+Progressive loading is when a page loads in batches. The simplest view of the page loads first, followed by progressively more detailed batches until the entire viewport has loaded.
+
+The primary batch should show a page's basic structure (the skeleton state versions of the container-based components), data-based text (the skeleton state version of text) and non-data text. Following batches can include images, content outside of the viewport, interactive (action-based) components, and data-based text.
+
+Not all items need a skeleton state and instead can be expressed as negative or white space until they load. For example, a 600 x 600px image can be shown as a 600 x 600px area of white space until the full image loads.
+
+<Row>
+<Column colLg={8}>
+
+![Example of a dashboard using skeleton states to demonstrate progressive loading.](/images/prog-loading-1.png)
```

Comment:
Should this set of images be larger since they are full screens? Do we have rules around when images are 12 col vs 8 col?

Reply:
We don't have strict rules. Typically header/lead images will be 12 col, but outside of that, it's undefined. I can play around with the image sizes here and see how it looks.

Reply:
Seems as if its a "full screen" type image then 12 col is better, where something more zoomed-in might make sense at 8?

Reply:
Agreed. I just added this to the style guide @janchild and I are putting together.